geospatial/src/Dem.cc

```diff
@@ -161,11 +167,34 @@ int Dem::Load(const std::string &_filename)
   if (this->LoadData() != 0)
     return -1;
 
-  // Set the min/max heights
-  this->dataPtr->minElevation = *std::min_element(&this->dataPtr->demData[0],
-      &this->dataPtr->demData[0] + this->dataPtr->side * this->dataPtr->side);
-  this->dataPtr->maxElevation = *std::max_element(&this->dataPtr->demData[0],
-      &this->dataPtr->demData[0] + this->dataPtr->side * this->dataPtr->side);
+  // Check for nodata value in dem data. This is used when computing the
+  // min elevation. If nodata value is not defined, we assume it will be one
+  // of the commonly used values such as -9999, -32768, etc.
+  // For simplicity, we will treat values <= -9999 as nodata values and
```

Comment:
is there documentation we can reference about these values?

Reply:
I haven't been able to find documentation on this other than this blog post https://rashms.com/gis/raster-file-resolution-extent-no-data-value-in-qgis/

Reply:
I found a reference to -9999 in the ArcGIS documentation: https://desktop.arcgis.com/en/arcmap/10.8/manage-data/raster-and-images/nodata-in-raster-datasets.htm

how do you feel about adding that link?

Reply:
Works for me, thanks for finding 5eadb52caf977cf1b891a8d00c00a1c48ea368b7